.github/workflows/compare_translations.py

```diff
@@ -17,7 +17,7 @@ def compare_translations(default_translation, other_translation, language):
 
 def check_translations():
     # Load default translation
-    with open("../../lang/en.json", "r", encoding="utf-8") as default_file:
+    with open("lang/en.json", "r", encoding="utf-8") as default_file:
```

Comment:
1. Follow the Google Python style guide especially in the DocString area. https://google.github.io/styleguide/pyguide.html
2. Make sure that the script is python `black`, `pydocstring`, `pylint`, `flake8` and `pycodestyle` compliant
4. Use the `countline.py` script in the `workflow` directory as a guide

![image](https://github.com/PalisadoesFoundation/talawa-admin/assets/16875803/eee20105-1c68-467d-b1cb-9371240a0678)

Reply:
Made the style of python code as mentioned
Thanks for the suggestion.